### ISSUE TITLE
Bug with POS option

### ISSUE THREAD

**Issue by Jean-Baptiste-Camps** [SupervisedStylometry/SuperStyl#55]
When I try to test it, I get:

```python
LookupError: 
**********************************************************************
  Resource averaged_perceptron_tagger not found.
  Please use the NLTK Downloader to obtain the resource:
  >>> import nltk
  >>> nltk.download('averaged_perceptron_tagger')
````

**Comment by Jean-Baptiste-Camps** [comment]
@floriancafiero !

**Comment by Jean-Baptiste-Camps** [comment]
Should be solved when #60 is tested and merged.
